.travis.yml

```diff
@@ -13,14 +10,18 @@ before_script:
 - pip install json-rpc
 - pip install psutil
 - pip install pycurl
-- cd docker
-- docker-compose up -d
-- docker-compose ps
-- cd ..
+- sudo apt-get -qq update
+- sudo /etc/init.d/postgresql stop
+- sudo apt-get install -y postgresql-contrib-9.4 postgresql-plpython-9.4
```

Comment:
I think this use is improperly although it can work. Refer [doc](https://docs.travis-ci.com/user/database-setup/#PostgreSQL). We should use `services`, and use `before_script` to create database, user and table.